source/github-events/on-pr-merge.ts

```diff
@@ -5,5 +5,5 @@ import observe from '../helpers/selector-observer.js';
 // This event ensures that the callback appears exclusively to the person that merged the PR and not anyone who was on the page at the time of the merge
 export default async function onPrMerge(callback: VoidFunction, signal: AbortSignal): Promise<void> {
 	await oneEvent('.js-merge-commit-button', 'click', {signal});
-	observe('.TimelineItem-badge .octicon-git-merge', callback, {signal, once: true});
+	observe('.TimelineItem-badge .octicon-git-merge', callback, {signal});
```

Comment:
Thank you! This means that the selector also matches things we don't expect and that callback is called multiple times on other elements. 

Probably not important at the moment since the PR view beta will come in the following months anyway.